Add feature display track length in an album

tungnk123 opened the pull request.

tungnk123 committed:
Translated using Weblate (Vietnamese)

Currently translated at 59.4% (173 of 291 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/vi/

tungnk123 committed:
Translated using Weblate (Vietnamese)

Currently translated at 12.5% (1 of 8 strings)

Translation: Gramophone/Fastlane (F-Droid / Google Play)
Translate-URL: https://hosted.weblate.org/projects/gramophone/fastlane/vi/

tungnk123 committed:
Add feature display track length in an album

tungnk123 committed:
Add feature display track length in an album

tungnk123 committed:
Add feature display track length in an album

nift4 committed:
some ttml stuff

nift4 committed:
copy pasting a sha sum is very hard

Commit:
Translated using Weblate (Czech)

Currently translated at 100.0% (8 of 8 strings)

Translation: Gramophone/Fastlane (F-Droid / Google Play)
Translate-URL: https://hosted.weblate.org/projects/gramophone/fastlane/cs/

Commit:
Translated using Weblate (Czech)

Currently translated at 100.0% (291 of 291 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/cs/

nift4 committed:
lol

nift4 committed:
ttml mvp

nift4 committed:
fix #393

nift4 committed:
lol

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (294 of 294 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/et/

nift4 committed:
fix bg timing junk info

nift4 committed:
fix bug

nift4 committed:
stop confusing left and right

nift4 committed:
get rid of () around x-bg

nift4 committed:
lol

nift4 committed:
test

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (294 of 294 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/uk/

nift4 commented:
CI is failing, can you rebase your branch?

nift4 committed:
cheap credits

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (295 of 295 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/uk/

nift4 committed:
preblend text color

rehork committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (295 of 295 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/pl/

tungnk123 committed:
Merge branch 'feature/issue-380-display-track-length-in-an-album' of https://github.com/tungnk123/Gramophone into feature/issue-380-display-track-length-in-an-album

tungnk123 commented:
Ok CI is completed

nift4 commented:
I spent some time testing this, however, the text disappears when using large font sizes in Android settings.
  <LinearLayout
        android:id="@+id/linearLayout"
        android:layout_width="0dp"
        android:layout_height="@dimen/grid_card_label_height"
the parent LinearLayout has an hardcoded height which is needed for fast scroll computation and that seems to be insufficient if there is a third line of text

nift4 committed:
fix #405

nift4 committed:
fix bug

nift4 committed:
beg for emails with crash logs + scroll lyrics to top when seeking to before first lyric

tungnk123 commented:
Could you pls suggest me some solution ?

nift4 commented:
maybe increasing grid_card_label_height, but I'm really not sure atm

txemaq committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (295 of 295 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/ca/

nift4 committed:
hack fix #376

nift4 committed:
delete leftover

nift4 committed:
add top/bottom space to new lyrics

nift4 committed:
lyric view: bring my own padding

nift4 committed:
scroll tweak

reindex-ot committed:
Translated using Weblate (Japanese)

Currently translated at 99.3% (293 of 295 strings)

Translation: Gramophone/App
Translate-URL: https://hosted.weblate.org/projects/gramophone/strings-xml/ja/

tungnk123 committed:
Fix bugs hidden text in grid_card_label

tungnk123 committed:
Fix bugs hidden text in grid_card_label

tungnk123 commented:
I increased grid_card_label_height and decreased text size a little bit. May be it works ^^

tungnk123 committed:
Merge branch 'feature/issue-380-display-track-length-in-an-album' of https://github.com/tungnk123/Gramophone into feature/issue-380-display-track-length-in-an-album
